firefox_puppeteer/tests/test_utils.py

```diff
@@ -0,0 +1,69 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from firefox_ui_harness import FirefoxTestCase
+
+
+class TestUtils(FirefoxTestCase):
```

Comment:
I'm not sure I understand.  Do you mean `get_all_urls_in_history`?  That is the only helper method I see, but you want me to put that in a class of its own?  Where should the class live, or is there an example of something similar?